mapper/tests.py

```diff
@@ -323,7 +326,19 @@ def test_form_correct_submission(self):
         self.assertRedirects(response, reverse('migration-list'))
 
         [migration] = MigrateSubscription.objects.all()
+        # Test object is created correctly
         self.assertEqual(migration.table_name, 'testtable1')
         self.assertEqual(migration.column_name, tables['testtable1'][0])
         self.assertEqual(migration.from_messageset, 1)
         self.assertEqual(migration.to_messageset, 2)
+
+        # Test object history is created correctly
+        [history] = LogEntry.objects.filter(object_id=migration.pk)
```

Comment:
Yup. This is a useful way of doing an assert on `len(things) == 1` as well as fetching the objects you want at the same time.